internal/convert/rs_config.go

```diff
@@ -0,0 +1,118 @@
+package convert
+
+import (
+	"strconv"
+
+	"github.com/Masterminds/semver/v3"
+	"go.mongodb.org/ops-manager/opsmngr"
+	"go.mongodb.org/ops-manager/search"
+)
+
+// RSConfig shared properties of replica sets, config servers, and sharded clusters
+type RSConfig struct {
+	Name           string           `yaml:"name,omitempty" json:"name,omitempty"`
+	FCVersion      string           `yaml:"featureCompatibilityVersion,omitempty" json:"featureCompatibilityVersion,omitempty"`
+	ProcessConfigs []*ProcessConfig `yaml:"processes,omitempty" json:"processes,omitempty"`
+	Tags           []string         `yaml:"tags,omitempty" json:"tags,omitempty"`
+	Version        string           `yaml:"version,omitempty" json:"version,omitempty"`
+}
+
+type patcher func(*ProcessConfig, string) *opsmngr.Process
+
+// path is a generic replica set patcher, you'll need to provide a function that
```

Comment:
[np] missing rest of comment

Reply:
not only that the method name had a typo, fixed